ElixrTalentConnect/ElixrTalentConnect/SignUP/SignUpViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  SignUpViewController.swift
+//  ElixrTalentConnect
+//
+//  Created by Shemil Tom on 15/01/24.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController ,UITableViewDelegate,UITableViewDataSource{
+    @IBAction func BackArrowTapped(_ sender: Any) {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    @IBAction func GoToSignInTapped(_ sender: Any) {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    private let viewModel = SignUpViewModel()
+    
+    @IBAction func SignUpTapped(_ sender: Any) {
+        let fullNameCell = tableView.cellForRow(at: IndexPath(row: viewModel.tableValues.firstIndex(of: .fullName) ?? 0, section: 0)) as? SignUPTableViewCell
```

Comment:
Instead of taking the values like this can we use a delegate method from cell which the controller can confirm and get the updates like that?